navigation/MainNavigation.js

```diff
@@ -115,6 +117,33 @@ const FeedStack = createStackNavigator(
   }
 );
 
+const SkillRequestsStack = createStackNavigator(
+  //TODO add the rest of skill request screens for conservationists here
+  {
+    SkillImpactRequest: SkillRequestScreen,
+  },
+  {
+    navigationOptions: {
+      tabBarLabel: 'Skilled Impact Requests',
```

Comment:
If above sounds fine, we can drop Requests from this label and just call it "Skilled Impact"